index.rst

```diff
@@ -1,62 +1,147 @@
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+.. sectnum::
 
-  Use the following syntax for sections:
+.. TODO: Delete the note below before merging new content to the main branch.
 
-  Sections
-  ========
+.. note::
 
-  and
+   **This technote is not yet published.**
 
-  Subsections
-  -----------
+   The description of the Campaign Management system in the first draft of DMTN-181 :cite:`DMTN-181` implies a few butler/pipeline middleware feature requests, and the processes and tooling for defining campaigns are expected to need new middleware support as well.
+   This technote will attempt to aggregate those feature requests, while outlining a few possible approaches for how campaign definition and management will interact with the middleware, based on the absence or presence of potential new middleware functionality.
 
-  and
+Introduction and Scope
+======================
 
-  Subsubsections
-  ^^^^^^^^^^^^^^
+DMTN-181 :cite:`DMTN-181` describes a Campaign Management system that provides tooling for organizing and tracking the multiple workflows that comprise a pipeline processing campaign.
+It also touches on what we will here call (for lack of a better term, so far) *Campaign Definition*: the tools and processes for determining the datasets, software, and configuration that will go into a campaign.
+The datasets are our primary concern here, and for these :cite:`DMTN-181` focuses on the concept of exposure *exclusion lists* represented as *data ID sets*.
+A data ID set is a table (at least conceptually) whose columns are data ID keys (butler data dimensions), with each data ID a row; :cite:`DMTN-181` proposes using ``git``-controlled JSON files as the source of truth for these, and uploading them to a table in the `Registry` database for use in `QuantumGraph` generation.
+It also proposes that these be permanent database tables for provenance purposes.
 
-  To add images, add the image file (png, svg or jpeg preferred) to the
-  _static/ directory. The reST syntax for adding the image is
+The processes and tools for creating and maintaining exclusion lists are considered out of scope for campaign management by :cite:`DMTN-181`, but it correctly notes that these must be assembled from many disparate inputs that cannot be assumed to be present in any single database (let along the `Registry` database), so generating exclusion lists from these inputs implicitly as part of the queries that back `QuantumGraph` generation is not viable.
+Defining those processes and tools fully is out of scope for this technote, too, but we will try to anticipate their demands on the middleware.
+That includes support for using the data repository as a place to store and query metrics, dataset annotations, and other criteria that play a role in building exclusion lists, as well as making the `Registry` database the source of truth for curated data ID sets (such as the exclusion lists themselves).
```

Comment:
Cite metrics and dataset annotations DMTNs.

Reply:
I've got specific subsections for these later where I was planning to make those references, but it doesn't hurt to cite them up here, too.